docusaurus/docs/reference/tunnelers/04-macos.mdx

```diff
@@ -41,3 +43,11 @@ The Enrollment Status will show **Enrolled** once it's completed. Once the ident
 
 ![image](/img/clients/mac-install7.png) 
 
+## Managing MFA with the CLI
+
+The tunneler CLI is installed automatically with the Ziti Desktop Edge for Mac. The tunneler CLI can be used to manage MFA separately from Ziti Desktop Edge for Mac.
+
+Using Terminal.app, complete the following workflow to register a time-based one-time password (TOTP) token generator with the OpenZiti Controller.

```

Comment:
Again, i don't think this will actually work. Did you try this and confirm it works?